bindings/gumjs/gumdukmodule.c

```diff
@@ -571,6 +573,27 @@ GUMJS_DEFINE_FUNCTION (gumjs_module_map_update)
   return 0;
 }
 
+GUMJS_DEFINE_FUNCTION (gumjs_module_map_copy_values)
+{
+  GumModuleDetails * details;
+  int i;
```

Comment:
This should be `guint` to match the type of `values->len`. Variables should also be declared chronologically, so this one should go after `values`.